outline.md

```diff
@@ -292,23 +292,15 @@ Instead let's use the data store to retrieve all of our blog posts:
 Now we should update our template to loop over each of our blog posts and render it:
 
     {{#each model as |post|}}
-    <article>
-      <h2>{{post.title}}</h2>
-      {{post.body}}
-    </article>
+      <article>
+        <h2>{{post.title}}</h2>
+        {{post.body}}
+      </article>
     {{/each}}
 
 The handlebars each helper allows us to enumerate over a list of items.  This should print out all of our blog posts to the page.  Let's check out our homepage in our browser again and make sure it worked.
 
-TODO: explain what it looks like when it worked and add a screenshot
-
-#### Acceptance testing
-
-With some user interaction added to our application we can now create an acceptance test.
-
-Let's test that the blog posts from our API show up on the homepage.
-
-TODO: Add acceptance test for our blog post models showing up
+![display all blog posts](https://s3.amazonaws.com/f.cl.ly/items/0e0w0u2b2o3d0r47301y/Screen%20Shot%202015-03-31%20at%201.45.32%20PM.png)
```

Comment:
My screenshot app Cloudly auto-uploads it for me to share. If you want to download and store it within the repo idm